src/components/Avatar/styled/avatar.ts

```diff
@@ -0,0 +1,49 @@
+import styled from 'styled-components/native';
+import { BaseProps } from '../../types';
+import { COLOR_BRAND, COLOR_WHITE } from '../../../styles/colors';
+
+interface Props extends BaseProps {
+    size?: string;
+    style?: object;
```

Comment:
same ^^
There is no need to redefine style.

Reply:
KK